pkg/workload/schemachange/error_screening.go

```diff
@@ -1116,7 +1116,7 @@ SELECT count(*) FROM %s
 }
 
 var (
-	regexpUnknownSchemaErr = regexp.MustCompile(`unknown schema \[\d+\]`)
+	regexpUnknownSchemaErr = regexp.MustCompile(`unknown schema "\[\d+]"`)
```

Comment:
just want to confirm that we're trying to match something like `unknown schema "[1]"` ?

Reply:
Yes the errors have the form: unknown schema "[1]". I'll add a comment to make it clear